lib/galaxy/schema/schema.py

```diff
@@ -689,14 +752,27 @@ class HDCASummary(HistoryItemCommon):
         title="Job Source Type",
         description="The type of job (model class) that produced this dataset collection. Used to track the state of the job.",
     )
+    job_state_summary: Optional[HDCJobStateSummary] = Field(
```

Comment:
Just to clarify this here, it is optional because not all collections are created through jobs. You can just POST a request to create a collection from existing datasets, like you do with the list builders.